src/pages/history/History.jsx

```diff
@@ -1,10 +1,35 @@
+import { Button, Empty } from "../../components";
+import { useVideos } from "../../context/videos/Context";
 import { useDocumentTitle } from "../../hooks";
```

Comment:
Liked the use of useDocumentTitle custom hook, to get document titles dynamically